### PR TITLE
Apply delete mode for confirm modal on deleting

### PR DESCRIPTION
**Pull request by yusukefs**
## What?

app-common をアップデートし各種削除時の確認モーダルで削除用のスタイルを設定

- データベース削除
- レコード削除
- ファイル削除

## Why?

削除確認モーダルのスタイルをわかりやすくしたい

## See also [Optional]

- Issue: https://github.com/dataware-tools/dataware-tools/issues/27
- app-common PR: https://github.com/dataware-tools/app-common/pull/86

## Screenshot or video [Optional]

スクリーンショット 2021-08-13 16 00 04
スクリーンショット 2021-08-13 16 00 12
スクリーンショット 2021-08-13 16 00 21

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/128#" title="Author self-approved">yusukefs</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-data-browser-next).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-data-browser-next/blob/master/OWNERS)~~ [yusukefs]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by WatanabeToshimitsu — APPROVED** [review]
lgtm

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/128#pullrequestreview-730268243" title="Approved">WatanabeToshimitsu</a>*, *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/128#" title="Author self-approved">yusukefs</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-data-browser-next).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-data-browser-next/blob/master/OWNERS)~~ [WatanabeToshimitsu,yusukefs]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
